src/blog/2023/09/dashboard-notebook-layout.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Dynamic Markdown, Tables & Notebooks with Dashboard 2.0
+subtitle: A dive into the new features available in 0.4.0 - The "Notebook" Layout and new dynamic Markdown & Table widgets.
+description: A dive into the new features available in 0.4.0 - The "Notebook" Layout and new dynamic Markdown & Table widgets.
+date: 2023-09-06
+authors: ["joe-pavitt"]
+image: "/blog/2023/09/images/db-notebook-tile.png"
+tags:
+    - posts
+    - releases
+    - community
+---
+
+Whilst we're still busy backporting through the existing Dashboard 1.0 features, we did want to highlight some new features we've built in Dashboard 2.0. 
+
+<!--more-->
+
+In our v0.4.0 release, we've introduced a new "Notebook" layout, alongside a new Table widget.
+
+The Notebook layout is designed to allow users to create Dashboards structured like a Notebook (most often seen with the likes of [Jupyter Notebooks](https://jupyter.org/) or [ObservableHQ](https://observablehq.com/)).
+
+Here we will deepdive into the Notebook layout, and show how, alongside our new **Markdown Node** ([docs](https://flowforge.github.io/flowforge-nr-dashboard/nodes/widgets/ui-markdown.html)), **Table Node** ([docs](https://flowforge.github.io/flowforge-nr-dashboard/nodes/widgets/ui-table.html)) and others, it's becoming easier to create dynamic and interactive Dashboards.
+
+_Note: If you're not familiar with Markdown, it's a simple markup language that allows you to format text. You can learn more about it [here](https://www.markdownguide.org/cheat-sheet/)._
+
+## Dashboard Hierarchy
+
+As a quick introductory note ahead of our below guide, each Dashboard is structured accordingly:
+
+- **Widget**: An individual functional block, e.g. button, chart, slider
+- **Group**: A collection of widgets that render together
+- **Page**: A single page/tab in your Dashboard. Each page can have it's own Layout, in this case we'll use "Notebook"
+- **UI**: Contains a collection of pages, deployed from Node-RED, provides the basic side navigation to switch between Pages.
+
+## Building a Notebook
+
+![Example Notebook created in Dashboard](./images/db-notebook-example.png)
+
+To get started, drop your first widget (in this case, we'll add a `ui-markdown`) onto the Node-RED canvas. This in turn will prompt us to create our first Group/Page/Dashboard which we can name & configure accordingly.
+
+Let's add the following Markdown to our first widget:
+
+```md
+# Markdown Content
+
+Here we can render dynamic Markdown content that is
+easily _styled_.
+
+{% raw %}
+We can inject `msg.payload`. For example, here is a
+timestamp updating every second: {{ msg.payload }}
```

Comment:
This is a mustache templating feature. So the markdown widget supports mustache it seems?

Reply:
You would be correct: https://flowforge.github.io/flowforge-nr-dashboard/nodes/widgets/ui-markdown.html

Reply:
We thank @Steve-Mcl for that excellent addition ([PR](https://github.com/flowforge/flowforge-nr-dashboard/pull/126))